src/main/java/com/direwolf20/buildinggadgets/common/world/FakeRenderWorld.java

```diff
@@ -43,6 +37,7 @@ public IFluidState getFluidState(BlockPos pos) {
         return null;
     }
 
+    /* TODO remove unused?
```

Comment:
Use or remove 👍

Reply:
That class already exists with these obsolete Methods in 1.12... As parts of the rendering code are still uncomnented (right?) I'm not sure if I've missed something.
=> I need to look at 1.12...

Reply:
Confused? lol